src/CommonITILObject.php

```diff
@@ -8454,7 +8454,10 @@ public function manageValidationAdd($input)
                     $add_done = false;
                     foreach ($validations_to_send as $user) {
                         // Do not auto add twice same validation
-                        if (!$validation->alreadyExists($values[$self_fk], $user)) {
+                        if (
+                            !$validation->alreadyExists($values[$self_fk], $user)
+                            || isset($input['_rule_process'])
+                        ) {
```

Comment:
> It's impossible to add a new validation via a business rule on a ticket when one already exists. This is blocked by the `CommonITILValidation::alreadyExists()` function.

Do we really want to remove this blockage and allow several identical validations?